src/Entity/Package.php

```diff
@@ -328,6 +344,31 @@ public function removeReferenceNumber()
     {
         $this->referenceNumber = null;
     }
+    
+    /**
+     * @return ReferenceNumber
+     */
+    public function getReferenceNumber2()
+    {
+        return $this->referenceNumber2;
+    }
+
+    /**
+     * @param ReferenceNumber $referenceNumber
+     *
+     * @return Package
+     */
+    public function setReferenceNumber2(ReferenceNumber $referenceNumber)
+    {
+        $this->referenceNumber2 = $referenceNumber;
+
+        return $this;
+    }
+
+    public function removeReferenceNumber2()
```

Comment:
I agree with you, the Shipment class doesn't have the remove method. and you can effectively remove it by setting it to null. I was just making the same methods available to referenceNumber2 as that were on original referenceNumber.